### PR TITLE
Document PLEXIL version needed for installation

### PR DESCRIPTION
**Pull request by kmdalal**
This is just a documentation change, for the corresponding pull request in ow_autonomy: https://github.com/nasa/ow_autonomy/pull/11

### CODE REVIEW

**Review by lanseafood — APPROVED** [review]
👍🏻

**Comment by kmdalal** [comment]
NOTE:  path_planning_commander.py was inadvertently included in the PR branch (OCEANWATER-496).  This is because this branch was forked from master and not melodic-devel, and master is not merged into melodic-devel.    It can be ignored, as it's already a released change.

File: ow_lander/scripts/path_planning_commander.py [hunk 1]
```diff
@@ -33,9 +33,9 @@ def __init__(self):
     robot = moveit_commander.RobotCommander()
     scene = moveit_commander.PlanningSceneInterface()
 
-    move_arm = moveit_commander.MoveGroupCommander("arm")
-    move_limbs = moveit_commander.MoveGroupCommander("limbs")
-    move_grinder = moveit_commander.MoveGroupCommander("grinder")
+    move_arm = moveit_commander.MoveGroupCommander("arm", wait_for_servers=15.0)
```
Comment (mogumbo) [inline ow_lander/scripts/path_planning_commander.py:36]: This is confusing me. The wait_for_servers pieces are not in melodic-devel, which implies they were either added as a hotfix to master but not melodic-devel or they were included in master at release time and later removed from melodic-devel. If the latter, we should not include this change.

Comment (kmdalal) [reply ow_lander/scripts/path_planning_commander.py]: After the merges (master -> melodic-devel -> OW-496) this diff is gone and everything looks as expected!

**Review by mogumbo — APPROVED** [review]

**Pull request merged**